### PR TITLE
Fixes and reorder of installation page

### PR DESCRIPTION
**Pull request by tbruno**
This PR makes the following changes:
- Clarify that user also needs to enable policy enforcement and telemetry on the prerequisites part
- Add instruction to install manifest. 
- Reorder to place istio 1.5.x before 1.4.x

### CODE REVIEW

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -164,8 +152,22 @@ spec:
...
++
+`istioctl manifest apply -f <manifest-file.yaml>`
++
+
+
+=== Install Istio 1.4.x
+
+To install and configure Istio 1.4.x for Anypoint Service Mesh:
 
+. Download Istio using the https://istio.io/docs/setup/[Istio Documentation].
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:166]: maybe?

```suggestion
. Download `istioctl` following the https://istio.io/docs/setup/[Istio Documentation].
```

Comment (tbruno) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:166]: Not sure TBH, I think for customers that don't have previous Istio knowledge, it is better to download the folder and get the samples and tools.

Comment (dosten) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:166]: Sure, can we change only `using` -> `following`? seems better to me

Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:166]: Leaving as is based on Tomas's confirmation.

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**